src/components/notifications/HostedSMS.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div class="mb-3">
+        <label for="hostedsms-username" class="form-label">{{ $t('hostedsmsAPIUser') }}</label>
+        <input id="hostedsms-username" v-model="$parent.notification.hostedsmsUsername" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="hostedsms-key" class="form-label">{{ $t('hostedsmsAPIPassword') }}</label>
+        <HiddenInput id="hostedsms-key" v-model="$parent.notification.hostedsmsPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="hostedsms-phone-number" class="form-label">{{ $t("hostedsmsPhoneNumber") }}</label>
+        <input id="hostedsms-phone-number" v-model="$parent.notification.hostedsmsPhoneNumber" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="hostedsms-sender-name" class="form-label">{{ $t("hostedsmsSenderName") }}</label>
+        <input id="hostedsms-sender-name" v-model="$parent.notification.hostedsmsSenderName" type="text" minlength="3" maxlength="11" class="form-control">
```

Comment:
You have validated the `hostedsmsSenderName` to 3..11 chars.
Is this intentional? (as in required by their docs)

Some names might not fit this shema...

Reply:
Yes, documentation don't inform about the minimum and maximum number of characters. I'll remove it for safety.